portal/app/farm_updater/sonnen_api.py

```diff
@@ -72,7 +71,5 @@ def set_battery_action(mode='charge', value='0'):
     sonnen_api = SonnenApiInterface()
     control = sonnen_api.manual_mode_control(mode, value)
     print('control: ', control)
-    if int(control['ResponseCode']) != 0:
+    if control is None or int(control['ResponseCode']) != 0:
```

Comment:
If this whole check is just to re-print `control` then we can completely remove it. `control` is being printed on the line above already.

Reply:
Of course!